src/components/Security.js

```diff
@@ -174,7 +167,7 @@ class Security extends Component {
           <div id="register-webauthn-tokens-area" className="table-responsive">
             {securitykey_table}
             <div className="register-authn-buttons">
-             {platformAuthenticatorButton}
+            {platformAuthenticatorButton}
               <button
                 id="security-webauthn-button"
                 className={this.state.className}
```

Comment:
It looks like `this.state.className` won't be available before `PublicKeyCredential.isUserVerifyingPlatformAuthenticatorAvailable()` return or if `window.PublicKeyCredential` is false.